datasketch/storage.py

```diff
@@ -10,7 +10,7 @@
     redis = None
 
 
-def ordered_storage(config, name=None):
+def ordered_storage(config, name=None, buffer_size=50000):
```

Comment:
Same here. The buffer size is better specified when opening an insertion session.

Reply:
moved to insert_session argument